### PR TITLE
Mention official packages in the readme

### PR DESCRIPTION
**Pull request by fregante**
Re: https://github.com/sindresorhus/gh-got/issues/39#issuecomment-705851107

How does this look?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 > Convenience wrapper for [Got](https://github.com/sindresorhus/got) to interact with the [GitHub API](https://developer.github.com/v3/)
 
+If you're not already using Got, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) library.
```
Comment (sindresorhus) [inline readme.md:5]: How about this?

```suggestion
Unless you just need this for a single API call, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) package instead.
```

Comment (sindresorhus) [reply readme.md:5]: Can you include the GraphQL package too?

Comment (fregante) [reply readme.md:5]: > this for a single API call

Arguably you shouldn't use got for a single call either

Comment (fregante) [reply readme.md:5]:
```suggestion
Unless you're already using Got, you should probably use GitHub's own [@octokit/rest.js](https://github.com/octokit/rest.js) or [@octokit/graphql.js](https://github.com/octokit/graphql.js) packages instead.
```

**Pull request merged**